.github/PULL_REQUEST_TEMPLATE/preview-feature-pr.md

```diff
@@ -1,114 +1,56 @@
-    - [ ] Product Documentation
-      - [ ] A ![documentation-product-label](https://user-images.githubusercontent.com/85414302/154672508-5ac50eb7-67f8-4cdf-92b9-fe5fcbd93b14.png) label has been added to the pull request.
-- [ ] No
-</details>
-
-
-<details open><summary>🧪Unit Testing🧪</summary>
+      ``` xml
+      Syntax: preview/feature/<issue-num>-<description>
+      Example: preview/feature/123-my-preview-feature
+      ```
+    </details>
 
-My change requires unit tests to be written.
-- [ ] Yes
-- [ ] No
-</details>
+- [ ] The **_head(source)_** branch for this pull request is created from a **_preview release_** branch with the correct naming syntax in the following example.
+  <details closed><summary>Example</summary>
 
+    ``` xml
+    Syntax: preview/v<major>.<minor>.<patch>-preview.<prev-num>
+    Example: preview/v1.2.3-preview.4
+    ```
+  </details>
 
-<details open><summary>🧪Manual Testing🧪</summary>
+- [ ] The **_base(target)_** branch for this pull request is a **_preview release_** branch with the correct naming syntax in the following example.
+  - A **_base(target)_** branch is the branch that is a source branch being merged into it.
+    <details closed><summary>Example</summary>
 
-I have manually tested my changes using the project named **_VelaptorTesting_**.
-- [ ] Yes
-- [ ] No
-</details>
+      ``` xml
+      Syntax: preview/v<major>.<minor>.<patch>-preview.<prev-num>
+      Example: preview/v1.2.3-preview.4
+      ```
+    </details>
 
+💡For more info on branching, refer to the projects [branching documentation](../../Documentation/Branching.md).
 
-<h2 style="font-weight:bold" align="center">✅Code Review Checklist✅</h2>
+- [ ] Pull request title matches the title of the associated issue.
 
-<!-- Go over all of the following points, and put an `x` in all the boxes that apply. -->
-<!-- If you're unsure about any of these, don't hesitate to ask.  We are here to help! -->
-- [ ] Pull request title matches the title of the linked issue.
 - [ ] Associated issue exists and is linked to this pull request.
   - One issue per pull request.
+
+- [ ] The labels attached to this PR match the labels attached to the associated issue.
+
 - [ ] My code follows the coding style of this project.
   - The style is enforced by the *.editorconfig* files in the project and displayed as warnings.  If there is an edge case with a coding style that should be ignored or changed, reach out, and let's discuss it.
+
 - [ ] A ![preview-label](https://user-images.githubusercontent.com/85414302/150838564-33f6044b-55f9-4dd9-8783-1d739de9d92f.png) label has been added to the pull request.
+
+- [ ] I have manually tested my code changes to the best of my ability.
+
 - [ ] All tests passed locally.
   - This is required because unit tests might pass locally but not in the CI environment during the status check process or vise-versa.
```

Comment:
Misspelled vice
```suggestion
  - This is required because unit tests might pass locally but not in the CI environment during the status check process or vice-versa.
```